### PR TITLE
Update table with sdk features

### PR DESCRIPTION
**Pull request by marcduiker**
Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added workflow checks for Java, Go and JS
Added Actor check for Rust

## Issue reference

#4096

### CODE REVIEW

**Review by mikeee — APPROVED** [review]
lgtm

File: daprdocs/content/en/developing-applications/sdks/_index.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ Select your [preferred language below]({{< ref "#sdk-languages" >}}) to learn mo
 |----------|:------|:----------:|:-----------:|:---------:|:---------:|
 | [.NET]({{< ref dotnet >}}) | Stable | ✔ |  [ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/examples/AspNetCore) | ✔ | ✔ |
 | [Python]({{< ref python >}}) | Stable | ✔ | [gRPC]({{< ref python-grpc.md >}}) <br />[FastAPI]({{< ref python-fastapi.md >}})<br />[Flask]({{< ref python-flask.md >}})| ✔ | ✔ |
-| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot | ✔ | |
-| [Go]({{< ref go >}}) | Stable | ✔ | ✔ | ✔ | |
+| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot | ✔ | ✔ |
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/sdks/_index.md]:
```suggestion
| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot  <br /> Quarkus| ✔ | ✔ |
```
I suggest also have Quarkus given samples like this 
https://github.com/mcruzdev/pizza-quarkus/commit/13ad7f4009379a228e8281c9039b6172604d0909

Comment (msfussell) [reply daprdocs/content/en/developing-applications/sdks/_index.md]: Also worth adding a link to Java samples for Spring Boot?

Comment (marcduiker) [reply daprdocs/content/en/developing-applications/sdks/_index.md]: As far as I can see, there are no good starter docs on using either using Spring Boot (or Quarkus) within the docs (or the SDk repo). I'll create two issues in the js-sdk repo for this.

**Review by hhunter-ms — APPROVED** [review]
lgtm

**Pull request merged**